### PR TITLE
issues-42 Whats new in SeaQuery 0.26

### PR DESCRIPTION
**Pull request by ikrivosheev**
## PR Info

- Closes https://github.com/SeaQL/seaql.github.io/issues/42

### CODE REVIEW

**Comment by tyt2y3** [comment]
```js
6:48:53 PM: SyntaxError: /opt/build/repo/Blog/blog/2022-07-23-whats-new-in-seaquery-0.26.0.md: Unterminated JSX contents. (177:16)
6:48:53 PM:   175 | <h2 {...{"id":"community"}}>{`Community`}</h2>
6:48:53 PM:   176 | <p>{`SeaQL is a community driven project. We welcome you to participate, contribute and together build for Rust's future.`}</p>
6:48:53 PM: > 177 |     </MDXLayout>
6:48:53 PM:       |                 ^
6:48:53 PM:   178 |   )
6:48:53 PM:   179 | };
6:48:53 PM:   180 | MDXContent.isMDXComponent = true
```

Not sure what this means @billy1624

**Comment by tyt2y3** [comment]
Looks good so far! I think sea-query 0.26 has a lot of new features / improvements worth mentioning.
Actually we can also include a few things from 0.25, I think `CASE WHEN` and `DISTINCT ON` are really great features too.

**Comment by ikrivosheev** [comment]
> Looks good so far! I think sea-query 0.26 has a lot of new features / improvements worth mentioning.
> Actually we can also include a few things from 0.25, I think `CASE WHEN` and `DISTINCT ON` are really great features to introduce too.

@tyt2y3 sorry, this is draft PR)

File: Blog/blog/2022-07-23-whats-new-in-seaquery-0.26.0.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+                    </a>
+                    <div class="avatar__intro">
+                        <div class="avatar__name">
+                            Chris Martin
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+        </div>
```
Comment (billy1624) [inline Blog/blog/2022-07-23-whats-new-in-seaquery-0.26.0.md:90]: Hey @ikrivosheev, removing this empty line should fix the build error

Comment (ikrivosheev) [reply Blog/blog/2022-07-23-whats-new-in-seaquery-0.26.0.md]: Thank you!

**Comment by ikrivosheev** [comment]
@billy1624 can you help me, whats happen?)

**Comment by billy1624** [comment]
Hey @ikrivosheev, I updated the build script. Please merge latest master into current branch :)

**Comment by billy1624** [comment]
I think we should bump the date of this blog post to sometime after https://www.sea-ql.org/blog/2022-07-30-engineering/

File: Blog/blog/2022-07-23-whats-new-in-seaquery-0.26.0.md [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+```
+
+## SeaQL Improvements
+
+- [[#353](https://github.com/SeaQL/sea-query/pull/353)] Support `LIKE ... ESCAPE ...`  expression  
+- [[#306](https://github.com/SeaQL/sea-query/pull/306)] Move `escape` and `unescape` string to backend
+- [[#365](https://github.com/SeaQL/sea-query/pull/365)] Add method to make a column nullable
+- [[#348](https://github.com/SeaQL/sea-query/pull/348)] Add `is` & `is_not` to Expr
+- [[#349](https://github.com/SeaQL/sea-query/pull/349)] Add `CURRENT_TIMESTAMP` function
+- [[#345](https://github.com/SeaQL/sea-query/pull/345)] Add `in_tuple` method to Expr
```
Comment (billy1624) [inline Blog/blog/2022-07-23-whats-new-in-seaquery-0.26.0.md:84]: Why we have two `SeaQL Improvements` sections?

File: Blog/blog/2022-07-23-whats-new-in-seaquery-0.26.0.md [hunk 3]
```diff
@@ -0,0 +1,284 @@
...
+```
+
+## SeaQL Improvements
+- [[#266](https://github.com/SeaQL/sea-query/pull/266)] Insert Default
+- [[#324](https://github.com/SeaQL/sea-query/pull/324)] Make `sea-query-driver` an optional dependency
+- [[#334](https://github.com/SeaQL/sea-query/pull/334)] Add `ABS` function
+- [[#332](https://github.com/SeaQL/sea-query/pull/332)] Support `IF NOT EXISTS` when create index
+- [[#314](https://github.com/SeaQL/sea-query/pull/314)] Support different `blob` types in MySQL
+- [[#331](https://github.com/SeaQL/sea-query/pull/331)] Add `VarBinary` column type
+- [[#335](https://github.com/SeaQL/sea-query/pull/335)] `RETURNING` expression supporting `SimpleExpr`
```
Comment (billy1624) [inline Blog/blog/2022-07-23-whats-new-in-seaquery-0.26.0.md:187]: And here

**Review by billy1624 — COMMENTED** [review]
Should we merge it into one `Improvements` section?

File: Blog/blog/2022-07-23-whats-new-in-seaquery-0.26.0.md (hunk 2 above)
Comment (ikrivosheev) [reply Blog/blog/2022-07-23-whats-new-in-seaquery-0.26.0.md]: Yep! Done)

File: Blog/blog/2022-07-23-whats-new-in-seaquery-0.26.0.md (hunk 3 above)
Comment (ikrivosheev) [reply Blog/blog/2022-07-23-whats-new-in-seaquery-0.26.0.md]: Yep! Done)

**Comment by tyt2y3** [comment]
Awesome!

**Pull request merged**